pkg/messagebroker/kafka.go

```diff
@@ -106,11 +106,13 @@ func newKafkaProducerClient(ctx context.Context, bConfig *BrokerConfig, options
 	configMap := &kafkapkg.ConfigMap{
 		"bootstrap.servers":       strings.Join(bConfig.Brokers, ","),
 		"socket.keepalive.enable": true,
-		"retries":                 2,
-		"linger.ms":               0,
-		"request.timeout.ms":      30000,
-		"delivery.timeout.ms":     150000,
+		"retries":                 3,
+		"linger.ms":               100,
```

Comment:
This seems 
1. High, it will add 100ms delay with decent small traffic 
2. not required on small load, required for very high load

Reply:
set it to zero for now.